apps/sms/desktop-mock/navigator_moz_mobilemessage_shim.js

```diff
@@ -7,12 +7,12 @@
 
 *********************************************************** */
 (function(window) {
- 
-  if (!parent.navigator.mozMobileMessage) {
+
+  if (!opener || !opener.navigator.mozMobileMessage) {
```

Comment:
mm, good question, I don't really remember already :) IIRC it was intended for alwaysLowered window